gui/wxpython/image2target/ii2t_manager.py

```diff
@@ -1514,47 +1504,49 @@ def SaveGCPs(self, event):
         """
         self.GCPcount = 0
         try:
-            f = open(self.file["control_points"], mode="w")
-            # use os.linesep or '\n' here ???
-            f.write("# Ground Control Points File\n")
-            f.write("# \n")
-            f.write("# target location: " + self.currentlocation + "\n")
-            f.write("# target mapset: " + self.currentmapset + "\n")
-            f.write("#\tsource\t\t\ttarget\t\t\tstatus\n")
-            f.write("#\teast\tnorth\theight\teast\tnorth\theight\t(1=ok, 0=ignore)\n")
-            f.write(
-                "#----------------------------     ---------------------------     ---------------\n"  # noqa: E501
-            )
-
-            for index in range(self.list.GetItemCount()):
-                if self.list.IsItemChecked(index):
-                    check = "1"
-                    self.GCPcount += 1
-                else:
-                    check = "0"
-                coord0 = self.list.GetItem(index, 1).GetText()
-                coord1 = self.list.GetItem(index, 2).GetText()
-                coord2 = self.list.GetItem(index, 3).GetText()
-                coord3 = self.list.GetItem(index, 4).GetText()
-                coord4 = self.list.GetItem(index, 5).GetText()
-                coord5 = self.list.GetItem(index, 6).GetText()
+            with open(self.file["control_points"], mode="w") as f:
+                # use os.linesep or '\n' here ???
```

Comment:
```suggestion
```
As answered in the other PR, the answer should be to use '\n' and let Python in text mode (with universal newlines) handle this.